config/initializers/carrierwave.rb

```diff
@@ -0,0 +1,11 @@
+CarrierWave.configure do |config|
+  config.fog_credentials = {
+    provider: 'AWS',
+    aws_access_key_id: Rails.application.secrets.aws_access_key_id,
+    aws_secret_access_key: Rails.application.secrets.aws_secret_access_key,
+    region: Rails.application.secrets.aws_region
+  }
+  config.fog_directory  = 'flashcards-mkdev'
+  config.fog_public     = false
+  config.fog_attributes = { 'Cache-Control' => "max-age=#{365.day.to_i}" }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.